src/main/java/taintedmagic/common/network/PacketSashToggle.java

```diff
@@ -0,0 +1,36 @@
+package taintedmagic.common.network;
+
+import baubles.common.lib.PlayerHandler;
+import cpw.mods.fml.common.network.simpleimpl.IMessage;
+import cpw.mods.fml.common.network.simpleimpl.IMessageHandler;
+import cpw.mods.fml.common.network.simpleimpl.MessageContext;
+import io.netty.buffer.ByteBuf;
+import net.minecraft.entity.player.EntityPlayer;
+import net.minecraft.item.ItemStack;
+import taintedmagic.client.handler.SashServerHandler;
+import taintedmagic.common.items.equipment.ItemVoidwalkerSash;
+
+public class PacketSashToggle implements IMessage, IMessageHandler<PacketSashToggle, IMessage> {
+
+    @Override
+    public void fromBytes(ByteBuf buf) {
+        // do nothing
```

Comment:
idk how reliable MC network system is but wouldn't it be better to pass the new state explicitly to avoid potential desync?

Reply:
well it's either on or off i don't think that passing status will change anything

Reply:
well, having worked a bit with other network/server systems I'm used to always account for the possybly lost packets, but maybe here it is not necessary

Reply:
well if the packet is lost it will desync even if we send status

Reply:
yes, but it will resync next time

Reply:
it's tcp, so it will resync anyway.